repo-tools/tool-test-helper/generate

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import os
+from datetime import datetime, timedelta
+
+import click
+
+INITIAL_PLUGIN_CONTENTS = """version: 0.1
+downloads:
```

Comment:
Maybe mention that a tool can have a download or a runtime. The linter generate script doesn't include a download section (maybe it should?)